src/universal/subscriptions/GitHubRepoAddedSubscription.js

```diff
@@ -1,7 +1,7 @@
 // import storeDebugger from 'relay-runtime/lib/RelayStoreProxyDebugger';
 import {addGitHubRepoUpdater} from 'universal/mutations/AddGitHubRepoMutation';
 
-const AddedSubscription = graphql`
+const subscription = graphql`
```

Comment:
a subscription file is only ever going to have 1 subscription query, so i want to standardize the naming.